### ISSUE TITLE
Error while repairing files

### ISSUE THREAD

**Issue by KrazyKeiKei** [goatcorp/FFXIVQuickLauncher#1465]
### Update disclaimer

- [X] Yes, I have checked and my issue is not related to the game updating and plugins not working correctly.

### What did you do?

I kept on getting messages from the plugin, Mare Synchronos, that my game files needed to be repaired by XIV Launcher. Whenever I closed the game it got stuck on a black screen, forcing me to use task manager. I finally tried to address the issue. I launched XIVLauncher and immediately held shift to get the main window. I clicked the arrows next to the login button, then clicked on the option for repairing game files. I waited for my game files to be verified, but it kept on encountering an error, thus bringing me here. 

### Platform

Windows

### Relevant log output

```shell
System.Exception: System.IO.IOException: The process cannot access the file because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.InternalMove(String sourceDirName, String destDirName, Boolean checkHost)
   at XIVLauncher.Common.Patching.IndexedZiPatch.IndexedZiPatchIndexRemoteInstaller.WorkerSubprocessBody.<<-ctor>b__6_0>d.MoveNext() in C:\goatsoft\xl\XIVLauncher.Common\Patching\IndexedZiPatch\IndexedZiPatchIndexRemoteInstaller.cs:line 489
   at XIVLauncher.Common.Patching.IndexedZiPatch.IndexedZiPatchIndexRemoteInstaller.<WaitForResult>d__17.MoveNext() in C:\goatsoft\xl\XIVLauncher.Common\Patching\IndexedZiPatch\IndexedZiPatchIndexRemoteInstaller.cs:line 166
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Common.Patching.IndexedZiPatch.IndexedZiPatchIndexRemoteInstaller.<MoveFile>d__35.MoveNext() in C:\goatsoft\xl\XIVLauncher.Common\Patching\IndexedZiPatch\IndexedZiPatchIndexRemoteInstaller.cs:line 329
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at XIVLauncher.Common.Game.Patch.PatchVerifier.<MoveUnnecessaryFiles>d__82.MoveNext() in C:\goatsoft\xl\XIVLauncher.Common\Game\Patch\PatchVerifier.cs:line 294
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at XIVLauncher.Common.Game.Patch.PatchVerifier.<RunVerifier>d__83.MoveNext() in C:\goatsoft\xl\XIVLauncher.Common\Game\Patch\PatchVerifier.cs:line 460

Version: 6.3.16.0
Git Hash: 6.3.16-0-g020dde9
Context: PatchVerifier
OS: Microsoft Windows NT 6.2.9200.0
64bit? True
Addons Enabled? True
Auto Login Enabled? True
Language: English
LauncherLanguage: English
Game path: C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn
```

**Comment by NotNite** [comment]
Duplicate of #1464. Close the game and any tools related to FFXIV before starting a repair.

**Comment by KrazyKeiKei** [comment]
Nothing for the game was open except for rXIVLauncher itself 
![image](https://github.com/goatcorp/FFXIVQuickLauncher/assets/165012988/f4e664e0-28fc-4665-b482-d5d057796a16)
